app/main.py

```diff
@@ -202,17 +201,18 @@ def rabbit_status_callback(self, response):
             status = res.get('status')
             logger.info("Rabbit MQ health check response {}".format(status))
             if status == "ok":
-                logger.info('Setting status now')
+                logger.info("Checked rabbit health", health="ok")
                 self.rabbit_status = True
 
     def determine_ftp_status(self):
         try:
             self.ftp_status = False
             conn = self.ftp.get_connection()
             if conn:
+                logger.info("Checked FTP health", health="ok")
                 self.ftp_status = True
-        except Error as e:
-            logger.error("FTP error raised", error=str(e))
+        except FTPException as e:
+            logger.error("FTP exception raised", error=str(e))
```

Comment:
Maybe instead of "ftp exception raised" this should be more in the line of:

Reply:
in line of?